Apps/Playground/X11/App.cpp

```diff
@@ -54,18 +55,23 @@ namespace
         inputBuffer.reset();
         graphics.reset();
     }
-    
-    void InitBabylon(int32_t window, int width, int height, int argc, const char* const* argv)
+
+    void InitBabylon(Window window, int width, int height, int argc, const char* const* argv)
     {
         std::vector<std::string> scripts(argv + 1, argv + argc);
         std::string moduleRootUrl = GetUrlFromPath(GetModulePath().parent_path());
 
         Uninitialize();
 
         // Separately call reset and make_unique to ensure prior state is destroyed before new one is created.
-        graphics = Babylon::Graphics::CreateGraphics((void*)(uintptr_t)window, static_cast<size_t>(width), static_cast<size_t>(height));
+        GraphicsConfiguration graphicsConfig = GraphicsConfiguration();
+        graphicsConfig.windowPtr = (void*)(uintptr_t)window;
```

Comment:
Linux is a bit of a special case here. `Window` is defined as a `long unsigned int` rather than a pointer type, which makes our shared casting code not work for this case (clang does not allow us to static cast to a non pointer type)

Rather than